tool/jt.rb

```diff
@@ -823,7 +823,7 @@ def pr(*args)
     else
       PR.pr_push *args
       # To regularly update bb/master
-      PR.pr_update_master
+      PR.pr_update_master skip_upstream_fetch: true
```

Comment:
Another way to fix this would be to extract a `fetch_upstream` method and "cache" the result so it's only called once, like `remote_urls`.
This is fine too, but requires more coordination between pr_push/pr_update.